src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  let totalPrice = days * PRICE_PER_DAY;
```

Comment:
Consider using `const` for `totalPrice` instead of `let` since `totalPrice` does not need to be reassigned, it can be calculated with the discounts directly in the return statement.